bedrock/products/templatetags/misc.py

```diff
@@ -164,7 +164,10 @@ def vpn_monthly_price(ctx, plan=VPN_12_MONTH_PLAN, country_code=None, lang=None,
 
     available_plans = _vpn_get_available_plans(country_code, lang, bundle_relay)
     selected_plan = available_plans.get(plan, VPN_12_MONTH_PLAN)
-    amount = selected_plan.get("price")
+    price = float(selected_plan.get("price"))
+    currency = selected_plan.get("currency")
+    currency_locale = lang.replace("-", "_")
+    amount = format_currency(price, currency, locale=currency_locale)
```

Comment:
Note to self: need to figure out how to handle unknown locales here that throw an error:

e.g. `unknown locale 'ach'`

Reply:
Note that this is also an issue for Relay e.g. https://www-dev.allizom.org/ach/products/relay/ which is 500

Reply:
Isn't `price` here something like `"US$4.99"`? In which case, `float` is going to throw an error.

I'm also curious if you need to convert currency strings formatted with different delimiters (e.g. `"4,99"`?

[`babel.numbers.parse_decimal`](https://babel.pocoo.org/en/latest/api/numbers.html#babel.numbers.parse_decimal) seems helpful. Plus, having a decimal vs a float is good for currencies b/c floats can end up like `4.9999988889` or something.

Looks like `format_currency` can also just take a string, so maybe we can avoid all of that anyway and pass it directly without converting it first?

Reply:
What I have here seems to work OK, and from what I can tell `format_currency` seems to use a sensible number of default decimals